src/main/kotlin/graphics/scenery/volumes/DisplayRangeEditor.kt

```diff
@@ -0,0 +1,92 @@
+package graphics.scenery.volumes
+
+import graphics.scenery.ui.RangeSlider
+import net.miginfocom.swing.MigLayout
+import javax.swing.*
+import kotlin.math.max
+import kotlin.math.roundToInt
+
+/**
+ * GUI for editing the display range of volumes.
+ * Part of TransferFunctionEditor
+ */
+class DisplayRangeEditor(private val tfContainer: HasTransferFunction): JPanel(){
+
+    //RangeEditor
+    private val minText: JTextField
+    private val maxText: JTextField
+    private val rangeSlider: RangeSlider
+    private val minValueLabel: JLabel
+    private val maxValueLabel: JLabel
+
+    init {
+        this.layout = MigLayout(
+            "fill",
+            "[left, 10%]5[right, 40%]5[left, 10%]5[right, 40%]"
+        )
+        val title = BorderFactory.createTitledBorder("Display Range")
```

Comment:
Combine with assignment on the next line, also that one has an unnecessary `this`.